frontend/cypress/support/commands.js

```diff
@@ -0,0 +1,94 @@
+const signUpEndpoint = 'http://gitea.kitspace.test:3000/user/kitspace/sign_up'
```

Comment:
what does the folder name `support` indicate? should these tests maybe be called `gitea-api` or something like that?

Reply:
It's enforced by the Cypress directory structure. The support root file can be configured to be any other file, though I prefer leaving it as is.

Reply:
I see, that's fine then. What does it mean though just out of interest?

Reply:
It contains utility functions though instead of accessing them directly from the files having them in the `support` makes them accessible as properties of the `Cypress` object.